modules/cnf-topology-aware-lifecycle-operator-platform-update.adoc

```diff
@@ -0,0 +1,179 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-platform-update_{context}"]
+= Performing a platform update
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Update ZTP to the latest version. 
+* Provision one or more managed clusters with ZTP.
+* Mirror the desired image repository.
+* Log in as a user with `cluster-admin` privileges.
+* Create {rh-rhacm} policies in the Hub cluster.
+
+.Procedure
+
+. Create a `PolicyGenTemplate` CR for the platform update:
+.. Save the following contents of the `PolicyGenTemplate` CR in the `du-upgrade.yaml` file.
++
+.Example of `PolicyGenTemplate` for platform update 
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "du-upgrade"
+  namespace: "ztp-group-du-sno"
+spec:
+  bindingRules:
+    group-du-sno: ""
+  mcp: "master"
+  remediationAction: inform
+  sourceFiles: 
+    - fileName: ImageSignature.yaml <1>
+      policyName: "platform-upgrade-prep"
+      binaryData:
+        ${DIGEST_ALGO}-${DIGEST_ENCODED}: ${SIGNATRUE_BASE64} <2>
+    - fileName: DisconnectedICSP.yaml
+      policyName: "platform-upgrade-prep"
+      metadata:
+        name: disconnected-internal-icsp-for-ocp
+      spec:
+        repositoryDigestMirrors:
+          - mirrors:
+              - registry.example.com:5000/openshift4 <3>
+            source: quay.io/openshift-release-dev/ocp-release <4>
+    - fileName: ClusterVersion.yaml <5>
+      policyName: "platform-upgrade-prep"
+      metadata:
+        name: version
+      spec:
+        upstream: http://upgrade.example.com/images/upgrade-graph_stable-4.10
+    - fileName: ClusterVersion.yaml <6>
+      policyName: "platform-upgrade"
+      metadata:
+        name: version
+      spec:
+        channel: "stable-4.10"
+        upstream: http://upgrade.example.com/images/upgrade-graph_stable-4.10
+        desiredUpdate:
+          version: 4.10.4
+      status:
+        history:
+          - version: 4.10.4
+            state: "Completed"
+----
+<1> The `ConfigMap` CR contains the signature of the desired release image to update to.
+<2> Shows the image signature of the desired {product-title} release. To get the signature and replace the variables with the values, see step 2-7 of xref:../updating/updating-restricted-network-cluster.html#update-configuring-image-signature[Creating an image signature config map manually].
```

Comment:
This link doesn't have the steps of getting the image signature. Actually, the steps are only available in the OCP4.7 doc.
Instead of referencing to that link, in the **Preparing for the updates**, we should mention where/how to get the image signature and that is for later use in PGT for the platform upgrade. 
If the mirroring steps for **the mirror host does not have internet access** have been followed in **Mirroring the OpenShift Container Platform image repository** from https://deploy-preview-43890--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#installation-mirror-repository_ztp-deploying-disconnected, the image signature can be found from ${REMOVABLE_MEDIA_PATH}/mirror/config/<image_signature_file>
Also document the manual steps as a second option.

Reply:
@Missxiaoguo I updated this part, please check if that's what you meant.